tests/unit/scanners/test.javascript.js

```diff
@@ -170,6 +170,15 @@ describe('JavaScript Scanner', () => {
     expect(addonLinter.collector.warnings.length).toEqual(0);
   });
 
+  it('should parse as module .mjs files', async () => {
```

Comment:
Should we consider `.jsm` as module files too? (Sorry this does not necessarily belong here but I wanted to ask)

Reply:
> Should we consider .jsm as module files too? (Sorry this does not necessarily belong here but I wanted to ask)

@willdurand I don't think we have to parse `.jsm` as modules, technically the .jsm module files are not ESM modules (at least not at the moment), and so `import`/`export` are not technically valid statement if used in Firefox's `.jsm` module files. 

(If I'm not mistaken we added explicitly the `.jsm` file extension mainly to be sure we don't skip them in the validations rules, because they are used for our built in addons, which may include some of them as part of code used by the experimental APIs that privileged addon can embed, and to prevent other third party extension to be able to skip validation rules in some of their js code by just renaming them into .jsm files).